Solutions/Syslog/Hunting Queries/squid_malformed_requests.yaml

```diff
@@ -1,6 +1,8 @@
 id: edbeec9f-86b9-475d-8a42-cc7b95ad2baa
 name: Squid malformed requests
 description: |
+    'Malformed web requests can indicate network security devices. Hunting numerous requests from one source helps locate compromised hosts. Internal sites may need exclusion. Assumes default squid log format'
```

Comment:
'Malformed web requests are sometimes used for reconnaissance to detect the presence of network security devices. A large number of requests from a single source may indicate compromised hosts. Assumes default squid log format.'

Reply:
Resolved